src/main/java/com/rackspacecloud/metrics/queryservice/controllers/QueryController.java

```diff
@@ -129,6 +128,9 @@ public QueryResult grafanaQuery(
         List<QueryDomainOutput> outputs = new ArrayList<>();
 
         for (QueryResult.Result result : results) {
+            if (result.hasError()) {
```

Comment:
I wanted to look into this to see if the outside check was seeing if any of the singular results had errors but I am having trouble finding the javadoc for it. 

Which brings me to the point of we should probably look at upgrading the influx library to the latest one that supports influx 2.X